sw/device/lib/dif/dif_otp_ctrl.h

```diff
@@ -520,6 +520,23 @@ dif_result_t dif_otp_ctrl_dai_digest(const dif_otp_ctrl_t *otp,
                                      dif_otp_ctrl_partition_t partition,
                                      uint64_t digest);
 
+/**
+ * Checks if the digest value for the given partition has been computed. Once a
+ * digest has been computed for a partition, the partition is write-locked.
```

Comment:
Also could be a read-lock if the partition is secret ;)

Reply:
Good point. I clarified the comment.